app/controllers/admin/config/flags_controller.rb

```diff
@@ -19,8 +19,43 @@ def toggle
   def index
   end
 
+  def new
+  end
+
+  def edit
+  end
+
+  def create
+    with_service(Flags::CreateFlag) do
+      on_success do
+        Discourse.request_refresh!
+        render_json_dump(result.flag_instance)
```

Comment:
You don't have to, but I would suggest naming `flag_instance` just `flag` instead of having it being different between the Create/Update services